systemd/osdf-director.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description = Pelican service osdf-director
+After = network.target nss-lookup.target
+
+[Service]
```

Comment:
For all these systemd services - should we provide a sysconfig-style file to make overrides via env var easier?

Reply:
For systemd services, admins should be making changes by editing the config instead of setting environment variables. But sure, I can add a sysconfig file for knobs that can't be set otherwise; it would be especially useful if we could use that sysconfig file for both the systemd-based and the supervisord-based services.

Reply:
I added EnvironmentFiles in `/etc/sysconfig/SERVICE`.